* http://bugs.gentoo.org/show_bug.cgi?id=236740
  don't use paths from the host OS, we have much nicer stuff

--- Modules/Platform/UnixPaths.cmake
+++ Modules/Platform/UnixPaths.cmake
@@ -9,7 +9,9 @@
 # search types.
 LIST(APPEND CMAKE_SYSTEM_PREFIX_PATH
   # Standard
-  / /usr /usr/local
+  # / /usr /usr/local
+  "${CMAKE_INSTALL_PREFIX}" "${CMAKE_INSTALL_PREFIX}/.."
+  / /usr
 
   # CMake install location
   "${_CMAKE_INSTALL_DIR}"
@@ -21,36 +21,38 @@
 # List common include file locations not under the common prefixes.
 LIST(APPEND CMAKE_SYSTEM_INCLUDE_PATH
   # Windows API on Cygwin
-  /usr/include/w32api
+  # /usr/include/w32api
 
   # X11
-  /usr/X11R6/include /usr/include/X11
+  # /usr/X11R6/include /usr/include/X11
 
   # Other
-  /opt/local/include /usr/pkg/include
-  /opt/csw/include /opt/include  
-  /usr/openwin/include
+  # /opt/local/include /usr/pkg/include
+  # /opt/csw/include /opt/include  
+  # /usr/openwin/include
   )
 
 LIST(APPEND CMAKE_SYSTEM_LIBRARY_PATH
   # Windows API on Cygwin
-  /usr/lib/w32api
+  # /usr/lib/w32api
 
   # X11
-  /usr/X11R6/lib /usr/lib/X11
+  # /usr/X11R6/lib /usr/lib/X11
 
   # Other
-  /opt/local/lib /usr/pkg/lib
-  /opt/csw/lib /opt/lib 
-  /usr/openwin/lib
+  # /opt/local/lib /usr/pkg/lib
+  # /opt/csw/lib /opt/lib 
+  # /usr/openwin/lib
   )
 
 LIST(APPEND CMAKE_SYSTEM_PROGRAM_PATH
-  /usr/pkg/bin
+  # /usr/pkg/bin
+  "${CMAKE_INSTALL_PREFIX}/bin"
   )
 
 LIST(APPEND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES
-  /lib /usr/lib /usr/lib32 /usr/lib64
+  "${CMAKE_INSTALL_PREFIX}/lib" "${CMAKE_INSTALL_PREFIX}/../lib"
+  /lib /usr/lib /usr/lib32 /usr/lib64
   )
 
 # Enable use of lib64 search path variants by default.
